nupic/research/frameworks/pytorch/imagenet/mixins/lr_range_test.py

```diff
@@ -0,0 +1,111 @@
+# ------------------------------------------------------------------------------
+# Copyright (C) 2020, Numenta, Inc. All rights reserved.
+#
+# The information and source code contained herein is the
+# exclusive property of Numenta Inc.  No part of this software
+# may be used, reproduced, stored or distributed in any form,
+# without explicit written authorization from Numenta Inc.#
+# ------------------------------------------------------------------------------
+
+import os
+from copy import deepcopy
+
+from pandas import DataFrame
+from ray.tune.utils import flatten_dict
+
+from nupic.research.frameworks.pytorch.imagenet import mixins
+from nupic.research.frameworks.pytorch.lr_scheduler import LinearLRScheduler
+
+
+class LRRangeTest(mixins.LogEveryLoss):
```

Comment:
Is subclassing necessary? I think it's cleaner to try to keep all of the mixins orthogonal and let experiments choose which ones to opt into.

Reply:
It felt odd not to subclass. The idea of the test is to compare the training loss against the learning rate at every mini-batch step, so it makes sense to make sure they are logged.

I think I have an alternative idea, however. Let me know what you think of the `create_lr_test_experiment` function in the new changes.